.excludecoverage

```diff
@@ -6,3 +6,4 @@ tools/
 vendor/
 integration/
 testgen/
+openapi/
```

Comment:
dunno the right answer here - is there a reason to prefer `openapi/` as a top level directory over `src/coordinator/openapi`?

Reply:
I've put this under `src/coordinator/generated/assets/openapi/` now since it gets generated into embedded assets.